res/TensorFlowLiteRecipes/TConvSlice_000/test.recipe

```diff
@@ -0,0 +1,96 @@
+operand {
```

Comment:
Can you rename this to `Net_TConv_Slice_000` ?

Reply:
yes. I'll rename it. Thank you.

Reply:
@seanshpark @jinevening 
I've introduced 3 more tests. So now there are 4 tests:
```
Net_TConv_Slice_000 - test for zero spatial offset but spatial reduction in size
Net_TConv_Slice_001 - test for nonzero spatial offset and spatial reduction in size
Net_TConv_Slice_002 - test for channel-direction slice
Net_TConv_Slice_003 - test for original `same` padding and non-zero offset
```
Please let me know if we need more tests.